projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -28,58 +29,87 @@ import {
   filter,
   takeUntil,
   distinctUntilChanged,
+  catchError,
+  switchMap,
 } from 'rxjs/operators';
 
 import { Auth0ClientService } from './auth.client';
 import { WindowService } from './window';
 import { AbstractNavigator } from './abstract-navigator';
+import { connectableObservableDescriptor } from 'rxjs/internal/observable/ConnectableObservable';
```

Comment:
similar to the other comment. where is this being used?

Reply:
No, this was mistakenly added by VS Code while I was mistyping, I've removed it.